sdks/python/apache_beam/transforms/window.py

```diff
@@ -65,6 +65,21 @@
 from apache_beam.utils.windowed_value import WindowedValue
 
 
+__all__ = [
+    'TimestampCombiner',
+    'WindowFn',
+    'BoundedWindow',
+    'IntervalWindow',
+    'TimestampedValue',
+    'GlobalWindow',
+    'NonMergingWindowFn',
+    'GlobalWindows',
+    'FixedWindows',
+    'SlidingWindows',
+    'Sessions',
+    ]
```

Comment:
Or should we keep it Experimental for now (it's not an exposed concept in Java).